test/utils/configuration.test.ts

```diff
@@ -0,0 +1,18 @@
+import {expect} from 'chai'
+import configuration from '../../src/utils/configuration'
+
+describe('Configuration', () => {
+  it('parses valid configuration', () => {
```

Comment:
What about an invalid configuration format?

Reply:
I've realized invalid configuration probably doesn't matter all that much in the current context because all supported values have good default fallbacks.

I had an initial idea to use TypeScript to validate the shape of the loaded configuration, but that's not actually possible because typings are checked at compile not run time. There are some schema libraries out there we could use, but given we have fallbacks for all values you can use right now I'm not sure it's all that important.

As we build out our configuration more, we might want to consider `percy config validate` to allow people to check their files.